### PR TITLE
Added modal component

### PR DESCRIPTION
**Pull request by poojavishnoi**

### CODE REVIEW

File: components/Modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (bharati-21) [inline components/Modal/modal.html:6]: You can add the meta author and description tag as well.

File: doc/doc.html [hunk 2]
```diff
@@ -251,6 +251,46 @@ <h3>Example of Different Button Sizes</h3>
...
+  style="border:0; overflow:hidden;"
```
Comment (bharati-21) [inline doc/doc.html:636]: Kindly avoid inline styles.

**Review by bharati-21 — APPROVED** [review]
Looks good to merge!

**Review by nyyshaaa — APPROVED** [review]
Good to merge!

File: components/Modal/modal.html (hunk 1 above)
Comment (poojavishnoi) [reply components/Modal/modal.html:6]: Its actually for the reference like this html page will not show on the page.I'll add meta tag in landing and deoc page.Thank you.

File: doc/doc.html (hunk 2 above)
Comment (poojavishnoi) [reply doc/doc.html:636]: OKayy

**Pull request merged**